### PR TITLE
fix: accessibility enhancements

### PR DESCRIPTION
**Pull request by rodneylab**
Addresses #173

### CODE REVIEW

File: www/src/_layouts/docs.njk [hunk 1]
```diff
@@ -34,24 +34,28 @@ styles:
...
-              <a href="{{ link.href }}" {% if link.href === page.url %}aria-current="page"{% endif %}>{{ link.title }}</a>
-            </li>
-          {%- endfor -%}
-        </ul>
+        <nav aria-labelledby="side-nav-links-guides">
+          <span class="category" id="side-nav-links-guides" aria-hidden>{{ navLinksSorted.guides.label }}</span>
+          <ul role="list">
+            {%- for link in navLinksSorted.guides.links -%}
+              <li>
+                <a href="{{ link.href }}" {% if link.href === page.url %}aria-current="page"{% endif %}>{{ link.title }}</a>
```
[the remark this answers is not in the document]
Comment (rodneylab) [reply www/src/_layouts/docs.njk:42]: Thanks for reviewing. I'll step out here, happy for you to amend as necessary.

[the remark this answers is not in the document]
Comment (AleksandrHovhannisyan) [reply www/src/_layouts/docs.njk:42]: No you're fine! I probably shouldn't be the one reviewing/amending here. I just checked in on the ticket and noticed the PR and figured I'd take a quick peek. I'll let the core maintainers weigh in when they have time.

**Review by BenDMyers — CHANGES_REQUESTED** [review]
Howdy, @rodneylab! Thank you so much for this contribution.

One thing I've noticed: it seems as though the change in markup structure is causing a change in appearance — specifically, your deploy preview seems to have bullets for each of the list item leaf nodes.

This is the result of the following styles, which you can find in `docs.scss`

```scss
&__side-nav {
  // ...
  > li {
      // ...
      > ul {
        list-style: none;
    }
  }
}
```

This is generating the CSS selector `.docs__side-nav > li > ul` - which won't apply now that we have an extra `nav` layer. It might be worth reworking this selector to not require direct children.

**Comment by rodneylab** [comment]
> Just a quick question: Do we want to remove `aria-hidden`? This was my ill-informed recommendation in the original ticket, so apologies for the confusion around that.

I'm no accessibility expert, so yourself and others are likely better placed to answer that question.

**Comment by BenDMyers** [comment]
Sorry for the radio silence! This is a situation that I think could go either way, but my recommendation is to **remove** the `aria-hidden`.

**Comment by rodneylab** [comment]
> Sorry for the radio silence! This is a situation that I think could go either way, but my recommendation is to **remove** the `aria-hidden`.

not at all! Just made changes, hopefully all good.

**Pull request merged**